### ISSUE TITLE
Responsiveness and Accessibility 

### ISSUE THREAD

**Issue by jhawlc** [11160699/11160699.github.io#22]
Your images should resize if the pages resize
There should be alt text on each image to describe it.

Example: 

`<img src ="images/arms.jpg">`

Should be

`<img src ="images/arms.jpg" alt="Wrist Tech" width=20% height=10%>
`
